src/components/text-input/__tests__/__snapshots__/TextInput.test.js.snap

```diff
@@ -14,9 +14,6 @@ exports[`components/text-input/TextInput should render text input with descripti
       some description
     </div>
     <Tooltip
-      constrainToScrollParent={false}
```

Comment:
What's causing these default props to be dropped from the snapshots?

Reply:
the InjectIntl wrapped, just remove it and updated snapshots